fast64_internal/sm64/sm64_geolayout_bone.py

```diff
@@ -5,28 +5,30 @@
 from ..f3d.f3d_material import *
 
 enumBoneType = [
-	("Switch", "Switch (0x0E)", "Switch"), 
-	("Start", "Start (0x0B)", "Start"), 
-	("TranslateRotate", "Translate Rotate (0x10)", "Translate Rotate"), 
-	("Translate", "Translate (0x11)", "Translate"), 
-	("Rotate", "Rotate (0x12)", "Rotate"), 
-	("Billboard", "Billboard (0x14)", "Billboard"), 
-	("DisplayList", "Display List (0x15)", "Display List"), 
-	("Shadow", "Shadow (0x16)", "Shadow"), 
-	("Function", "Function (0x18)", "Function"), 
-	("HeldObject", "Held Object (0x1C)", "Held Object"), 
-	("Scale", "Scale (0x1D)", "Scale"), 
-	("StartRenderArea", "Start Render Area (0x20)", "Start Render Area"), 
-	("Ignore", "Ignore", "Ignore bones when exporting."), 
-	("SwitchOption", "Switch Option", "Switch Option"), 
-	("DisplayListWithOffset", "Display List With Offset (0x13)", 
-		"Display List With Offset"), 
+	("Switch", "Switch (0x0E)", "Switch"),
+	("Start", "Start (0x0B)", "Start"),
+	("TranslateRotate", "Translate Rotate (0x10)", "Translate Rotate"),
+	("Translate", "Translate (0x11)", "Translate"),
+	("Rotate", "Rotate (0x12)", "Rotate"),
+	("Billboard", "Billboard (0x14)", "Billboard"),
+	("DisplayList", "Display List (0x15)", "Display List"),
+	("Shadow", "Shadow (0x16)", "Shadow"),
+	("Function", "Function (0x18)", "Function"),
+	("HeldObject", "Held Object (0x1C)", "Held Object"),
+	("Scale", "Scale (0x1D)", "Scale"),
+	("StartRenderArea", "Start Render Area (0x20)", "Start Render Area"),
+	("Ignore", "Ignore", "Ignore bones when exporting."),
+	("SwitchOption", "Switch Option", "Switch Option"),
+	("DisplayListWithOffset", "Animated Part (0x13)", "Animated Part (Animatable Bone)"),
+	("CustomAnimated", "Custom Animated", "Custom Bone used for animation"),
+	("CustomNonAnimated", "Custom (Non-animated)", "Custom geolayout bone, non animated"),
 ]
 
+animatableBones = {"DisplayListWithOffset", "CustomAnimated"}
```

Comment:
```suggestion
animatableBoneTypes = {"DisplayListWithOffset", "CustomAnimated"}
```
I assume, since those are from `enumBoneType` above

Reply:
yup i agree! done.